src/utils/get_time_since.rs

```diff
@@ -0,0 +1,45 @@
+use chrono::{DateTime, Utc};
+use leptos_fluent::tr;
+
+pub fn get_time_since(date_time: &DateTime<Utc>) -> String {
+  let now = Utc::now();
+
+  let years = now.years_since(*date_time).unwrap_or_default();
+  if years > 0 {
+    return tr!("years-ago", { "years" => years });
```

Comment:
You can pry remove all these return statements, and the `;` 's

Reply:
I gave it a try and it didn't work. I think that approach would only work if the conditionals were all part of one big `if/else if/else` expression.